bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types_test.go

```diff
@@ -142,6 +149,78 @@ func TestClusterValidate(t *testing.T) {
 			},
 			expectErr: true,
 		},
+		"returns_error_when_Ignition_fields_are_set_but_format_is_not_Ignition": {
```

Comment:
Nit:
Why use `_` here when the rest of the test cases use spaces?

Reply:
Ah yeah, this should probably be consistent with the rest of the codebase.

BTW, using `_` instead of ` `, makes test name greppable, as `go test` while printing converts ` ` to `_` anyway in test names.

Reply:
Changed to spaces for consistency.